src/AzureAppConfigurationOptions.ts

```diff
@@ -8,7 +8,13 @@ export const MaxRetries = 2;
 export const MaxRetryDelayInMs = 60000;
 
 export interface AzureAppConfigurationOptions {
-    selectors?: { keyFilter: string, labelFilter: string }[];
+    /**
+     * Selectors used to control what key-values are retrieved from Azure App Configuration.
+     *
+     * @property keyFilter: A filter that determines the set of keys that are included in the configuration provider, cannot be omitted.
+     * @property labelFilter: A filter that determines what label to use when selecting key-values for the the configuration provider, omitted for no label.
+     */
+    selectors?: { keyFilter: string, labelFilter?: string }[];

```

Comment:
Can you use the same description that we have in [.NET provider](https://github.com/Azure/AppConfiguration-DotnetProvider/blob/75f3d6c186bbfb8c43e62f37207b1193e66e39e6/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs#L117-L136)? That summary has been iterated over a few times based on customer feedback and the same rules apply here too.

Reply:
sure, and maybe later I will probably create a type alias for `{ keyFilter: string, labelFilter?: string }`, e.g. `KeyValueSelector`.